src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,20 @@
-import React from 'react';
+import React, { Dispatch } from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
 
-export const TodoList: React.FC = () => (
+interface Props {
+  todos: Todo[];
+  onSelectedTodo: Dispatch<React.SetStateAction<Todo | null>>;
+  selectedTodo: Todo | null;
+  onSelectedUserId: Dispatch<React.SetStateAction<number>>;
+}
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  onSelectedTodo,
+  selectedTodo,
+  onSelectedUserId,
```

Comment:
The naming convention for your event handlers is not consistent with the conventions mentioned in the checklist. According to the [naming conventions for event handler functions](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364), the event handler prop should be named `onTodoSelect` and `onUserIdSelect` instead of `onSelectedTodo` and `onSelectedUserId`.